composer/trainer/trainer.py

```diff
@@ -722,6 +722,24 @@ def __init__(
                     To fix, please do not iterate over the dataloader before passing it into
                     the trainer."""))
 
+        # Evaluators
```

Comment:
See comment below, but is it possible to bind the evaluators to the state after Event.INIT (and leave this code where it was?)